tools/testbench/testbench.c

```diff
@@ -321,6 +321,11 @@ static int pipline_test(struct testbench_prm *tp)
 	int nsleep_time;
 	int nsleep_limit;
 
+	if (tp->quiet)
+		host_trace_level = LOG_LEVEL_INFO;
+	else
+		host_trace_level = LOG_LEVEL_DEBUG;
```

Comment:
you already can use `-d` to set `host_trace_level`, seems not very logical to have 2 arguments directly affect the same parameter. Besides `.quiet` is also already used to set `host_trace_level`:
https://github.com/thesofproject/sof/blob/cbcd51f9ea4e8f5f6e59d9a6dfb6f60b5bc05d94/tools/testbench/testbench.c#L485-L488
and then
https://github.com/thesofproject/sof/blob/cbcd51f9ea4e8f5f6e59d9a6dfb6f60b5bc05d94/tools/testbench/utils.c#L228-L230

Reply:
Yep, true that it is confusing and there would be lottery if use both. Maybe I should set default trace level to info and drop -q option (need to be removed from some scripts).